docs/pom.xml

```diff
@@ -42,6 +42,7 @@
     </properties>
 
     <modules>
+        <module>docs.api</module>
```

Comment:
As I tried command `mvn clean install -rf :org.eclipse.persistence.documentation -P docs` doesn't trigger `docs.api` module as there is different parent, than other `docs.` modules.
New `javadocs` module must be invoked directly like `mvn clean install -rf :org.eclipse.persistence.javadocs -P docs`?